### ISSUE TITLE
Map not visible due to 401 error with Mapbox key

### ISSUE THREAD

**Issue by ThomasG77** [openlayers/openlayers#9953]
Opening https://openlayers.org/en/latest/examples/layer-extent.html, we are unable to see the map

**To Reproduce**
1. Open your browser debugger
2. Go to 'https://openlayers.org/en/latest/examples/layer-extent.html'
3. See the following error in the console :
    ```
    TileJSON.js:104 GET https://api.tiles.mapbox.com/v3/mapbox.world-light.json?secure 401 (Unauthorized)
    ```

**Expected behavior**
The map should be visible. Change the API key to solve the issue.

**Comment by mike-000** [comment]
The v3 url isn't working even with a valid API key.  Other examples such as Layer Groups and TileJSON are also affected.  As well as an API key it looks like they will need to be changed to v4.

**Comment by marcjansen** [comment]
This should be fixed with #9964

**Comment by ThomasG77** [comment]
@marcjansen Thanks for the fix
